llama_index/indices/managed/colbert_index/base.py

```diff
@@ -121,12 +124,9 @@ def _build_index_from_nodes(self, nodes: Sequence[BaseNode]) -> IndexDict:
                 kmeans_niters=self.kmeans_niters,
             )
             indexer = Indexer(checkpoint=self.model_name, config=config)
```

Comment:
@logan-markewich their indexer

Reply:
Yea I suppose. It just feels very similar to BM25 since you can't add or delete data from it?

Reply:
oh they do have CRUD for index data: https://github.com/stanford-futuredata/ColBERT/blob/117098348e5196ede1c8e396c1c14f24d9a8754e/colbert/index_updater.py

the previous pr does not support it. I will add them for another pr

this pr is mostly for testing retrieval performance